src/SFA.DAS.Activities/IntegrityChecker/Fixers/NotFoundInElastic.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using SFA.DAS.Activities.IntegrityChecker.Dto;
+using SFA.DAS.Activities.IntegrityChecker.Interfaces;
+
+namespace SFA.DAS.Activities.IntegrityChecker.Fixers
+{
+    public class NotFoundInElasticFixer : IActivityDiscrepancyFixer
+    {
+        private readonly IElasticActivityDocumentRepository _elasticRepo;
+
+        public NotFoundInElasticFixer(IElasticActivityDocumentRepository elasticRepo)
```

Comment:
Can the file be renamed to match the type?

Reply:
Done